icub/conf/gazebo_icub_head.ini

```diff
@@ -23,34 +23,39 @@ device gazebo_controlboard
 #jointNames list
 jointNames neck_pitch neck_roll neck_yaw eyes_tilt left_eye right_eye
 name head
- 
+
 [COUPLING]
 eyes_vergence_control (4 5) (eyes_version eyes_vergence)
 
 #PIDs:
 # this information is used to set the PID values in simulation for GAZEBO, we need only the first three values
-[GAZEBO_PIDS]
-#head
-Pid0 100.0 7.0 0.2 9999 9999 9 9
-Pid1 100.0 7.0 0.2 9999 9999 9 9
-Pid2 100.0 7.0 0.2 9999 9999 9 9
-Pid3 250.0 0.0 0.1 9999 9999 9 9
-Pid4 250.0 0.0 0.1 9999 9999 9 9
-Pid5 250.0 0.0 0.1 9999 9999 9 9
-
-[GAZEBO_VELOCITY_PIDS]
-#head
-Pid0 500.0 2.0 0.2 9999 9999 9 9
-Pid1 500.0 2.0 0.2 9999 9999 9 9
-Pid2 500.0 2.0 0.2 9999 9999 9 9
-Pid3 250.0 0.0 0.1 9999 9999 9 9
-Pid4 250.0 0.0 0.1 9999 9999 9 9
-Pid5 250.0 0.0 0.1 9999 9999 9 9
+[POSITION_CONTROL]
+controlUnits  metric_units
+controlLaw    joint_pid_gazebo_v1
+kp 1.7453277777777776 1.7453277777777776 1.7453277777777776 4.3633194444444436 4.3633194444444436 4.3633194444444436
+kd 0.12217294444444443 0.12217294444444443 0.12217294444444443 0. 0. 0.
+ki 0.0034906555555555552 0.0034906555555555552 0.0034906555555555552 0.0017453277777777776 0.0017453277777777776 0.0017453277777777776
+maxInt 9999 9999 9999 9999 9999 9999
+maxOutput 9999 9999 9999 9999 9999 9999
+shift        
```

Comment:
Those parameters require the same number of columns of the others 
e.g. in this specific example six columns are required:  0.0 0.0 0.0 0.0 0.0 0.0